styles/components/card.css

```diff
@@ -0,0 +1,52 @@
+.card-sections {
+  display: grid;
+  overflow: hidden;
+  gap: 10px;
+  margin-top: 40px;
+  margin-bottom: 70px;
+}
+
+.card-set__bookmarklabel {
+  position: absolute;
+  top: 2px;
+  right: 15px;
+}
+.card-set__button {
+  color: transparent;
+}
+
+.card-set__bookmarklabel--booked {
+  fill: #ee632c;
+}
+
+.card-set {
+  display: grid;
+  gap: 10px;
+  border-style: outset;
+  background: #e6cbb5;
+  width: 96%;
+  justify-self: center;
+  align-self: center;
+  padding-left: 5px;
+  padding-right: 5px;
+  position: relative;
+}
+
+.card-set__tag-list {
+  display: grid;
```

Comment:
You might also use display: flex here, I think. This would make the tag list more dynamic, if there are shorter/longer tags. But it depends on the design, of course.

Reply:
oh danke, guter Punkt!